plugins/carbonetes-scan.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: carbonetes-scan
+spec:
+  homepage: https://github.com/carbonetes/kubectl-carbonetes-scan
+  shortDescription: Integrates container analysis directly into your cluster.
+  version: v1.1.0
+  description: |
```

Comment:
Please wrap around at ~80 chars, so that it prints nicely on wide and narrow terminals.